packages/openactive-integration-tests/config/default.json

```diff
@@ -53,8 +53,8 @@
     "business-to-business-tax-calculation-gross": null,
     "offer-overrides": null,
     "dynamic-payment": null,
-    "prepayment-optional": null,
-    "prepayment-unavailable": null,
+    "prepayment-optional": true,
+    "prepayment-unavailable": true,
```

Comment:
Based on the discussion, we should add an extra feature named `prepayment-required`, which is where the "not implemented" tests sit that use `TestOpportunityBookableFreePrepaymentUnspecified` and `TestOpportunityBookablePaidPrepaymentUnspecified`, and where the implemented tests sit that use `TestOpportunityBookablePaidPrepaymentRequired` and `TestOpportunityBookableFreePrepaymentRequired`

That test should be a prerequisite for both of these to run

Given this is the second time the idea of a prereq feature has come up (leasing was the first), I'll make a note to add prereqs as a feature of the test framework